packages/forklift-console-plugin/src/modules/Providers/utils/validators/provider/ovirt/validateOvirtUI.ts

```diff
@@ -0,0 +1,47 @@
+import { validateURL, ValidationMsg } from '../../common';
+
+export const validateOvirtUI = (providerUI: string | number): ValidationMsg => {
```

Comment:
can you make the name something with "link", my concern here is that `OvirtUI` sound like some UI component and not a link or a URL ...